UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/FileContentHelper.cs

```diff
@@ -67,6 +67,13 @@ public static void CheckProductFileContent(string inputFile, dynamic scsResponse
             Assert.True(fileContent[0].Contains(currentDate), $"Product File returned {fileContent[0]}, which does not contain expected {currentDate}");
             Assert.True(fileContent[1].Contains("VERSION"), $"Product File returned {fileContent[1]}, which does not contain expected VERSION.");
             Assert.True(fileContent[3].Contains("ENC"), $"Product File returned {fileContent[3]}, which does not contain expected ENC.");
+          
+            int rowNumber = new Random().Next(4, fileContent.Length-1);
+            var productData = fileContent[rowNumber].Split(",").Reverse();
+            var encryptionFlag = productData.ToList()[4];
+            string expectedEncryptionFlag = "1";
+            if (exchangeSetStandard == "s57") { expectedEncryptionFlag = "0";}
```

Comment:
Move `{ expectedEncryptionFlag = "0";}` to its on line and add some spacing i.e.

```c#
if (exchangeSetStandard == "s57")
{
      expectedEncryptionFlag = "0";
}

Reply:
I have made the required change.